### ISSUE TITLE
java.lang.NullPointerException when loading data types from Windows 10 21286 twinui.pdb

### ISSUE THREAD

**Issue by riverar** [NationalSecurityAgency/ghidra#2627]
**Describe the bug**
PDB load failure when attempting to load types from twinui.pdb/dll belonging to Windows 10 21286 (amd64).

**To Reproduce**
1. Download https://msdl.microsoft.com/download/symbols/twinui.dll/68C98C8F5ce000/twinui.dll
2. Download https://msdl.microsoft.com/download/symbols/twinui.pdb/68CA1F1C7B9561940582822784A3E0601/twinui.pdb
3. Open with Ghidra, skipping auto-analysis if configured/prompted
4. File > Load PDB
5. Select the aforementioned `twinui.pdb`, using either the `PDB Universal` or `PDB MSDIA engines`. Ensure restrictions are not selected (or minimally set to `Data types only`).

**Expected behavior**
PDB load to complete successfully or minimally with only warnings.

**Screenshots**
![image](https://user-images.githubusercontent.com/475132/103827487-f808aa80-502d-11eb-8659-943f93abd668.png)

**Environment**
```
java.lang.NullPointerException
	at ghidra.app.util.pdb.pdbapplicator.ArrayTypeApplier.applyArrayMsType(ArrayTypeApplier.java:132)
	at ghidra.app.util.pdb.pdbapplicator.ArrayTypeApplier.applyType(ArrayTypeApplier.java:111)
	at ghidra.app.util.pdb.pdbapplicator.ArrayTypeApplier.applyOrDeferForDependencies(ArrayTypeApplier.java:73)
	at ghidra.app.util.pdb.pdbapplicator.ArrayTypeApplier.apply(ArrayTypeApplier.java:62)
	at ghidra.app.util.pdb.pdbapplicator.PdbApplicator.processDataTypesSequentially(PdbApplicator.java:649)
	at ghidra.app.util.pdb.pdbapplicator.PdbApplicator.processSequentially(PdbApplicator.java:720)
	at ghidra.app.util.pdb.pdbapplicator.PdbApplicator.processTypes(PdbApplicator.java:235)
	at ghidra.app.util.pdb.pdbapplicator.PdbApplicator.applyTo(PdbApplicator.java:194)
	at pdb.LoadPdbTask.parseWithNewParser(LoadPdbTask.java:189)
	at pdb.LoadPdbTask$1.analysisWorkerCallback(LoadPdbTask.java:81)
	at ghidra.app.plugin.core.analysis.AutoAnalysisManager$AnalysisWorkerCommand.applyTo(AutoAnalysisManager.java:1700)
	at ghidra.app.plugin.core.analysis.AutoAnalysisManager$AnalysisTaskWrapper.run(AutoAnalysisManager.java:688)
	at ghidra.app.plugin.core.analysis.AutoAnalysisManager.startAnalysis(AutoAnalysisManager.java:788)
	at ghidra.app.plugin.core.analysis.AutoAnalysisManager.startAnalysis(AutoAnalysisManager.java:667)
	at ghidra.app.plugin.core.analysis.AutoAnalysisManager.startAnalysis(AutoAnalysisManager.java:632)
	at ghidra.app.plugin.core.analysis.AnalysisBackgroundCommand.applyTo(AnalysisBackgroundCommand.java:58)
	at ghidra.framework.plugintool.mgr.BackgroundCommandTask.run(BackgroundCommandTask.java:102)
	at ghidra.framework.plugintool.mgr.ToolTaskManager.run(ToolTaskManager.java:315)
	at java.base/java.lang.Thread.run(Thread.java:832)

---------------------------------------------------
Build Date: 2020-Nov-13 1112 EST
Ghidra Version: 9.2
Java Home: C:\Program Files\Java\jdk-14.0.2
JVM Version: Oracle Corporation 14.0.2
OS: Windows 10 10.0 amd64
US Citizen
```

**Comment by riverar** [comment]
I missed the point release in December, re-testing now.

**Comment by riverar** [comment]
This is not an issue in ghidra_9.2.1_PUBLIC_20201215. Closing.

**Comment by ghizard** [comment]
I'm glad it is working for you.
There were two issues fixed in 9.2.1. One was that the File->Load was broken in that it only would run PDB Universal, regardless of the user choice. The other issue fixed was a combination of adding a couple new data types and fixing a fallback mechanism that incorrectly used a void type instead of a void * type.
